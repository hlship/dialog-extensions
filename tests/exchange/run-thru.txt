give package to malory
talk to him
i
z
good
z
precautions
pass phrase
give case to malory
a house
weather
drop case
give it to him
