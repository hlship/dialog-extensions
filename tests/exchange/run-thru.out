

Another miserable night in the London fog and rain. You have the package, and
Malory is your contact... but can he be trusted?

Foggy Alley
You would be hard pressed to find a more cliche location for the drop off than
this deserted alley. Fog billows in from somewhere, and a lone lamppost sheds
minimal light. The sounds of bustling London are distant.

Malory, your contact, stands just outside the weak cone of light from the
lamppost.

> give package to malory
Protocol dictates that important assets should not be released until the
responsible agent confirms security precautions.

> talk to him
"Malory" is all you say. We're taught to keep these exchanges brief.

"Well," asks Malory, "do you have the package?" Always a bit too direct, this
one, everything's so urgent, no sense of panache.

You could say all in good time.

> i
Malory grows impatient. "The package!" he insists.

You have a briefcase.

You could say all in good time.

> z
Malory raises his voice a hair. "A simple 'yes' or 'no' would do."

A moment slips away.

You could say all in good time.

> good
"All in good time, Malory. There's a protocol to these things."

"Home Office was very clear," he responds, "there's a timetable to keep."

You could ask if all precautions have been made.

> z
A moment slips away.

You could ask if all precautions have been made.

> precautions
"Fine, then. On to business," you begin, "have all proper precautions been
taken?"

Malory gives you a hard stare. "Damn it, man. This is important--what are you
playing at?"

You could ask whether the safe house is ready or ask for the pass phrase.

> pass phrase
"I'm certainly not handing over something this valuable without the proper pass
phrase," you announce.

Malory's eyes flash left and right, and he leans in close. He whispers the
correct pass phrase almost inaudibly. You're somewhat relieved, but you still
have your orders.

You could ask whether the safe house is ready or comment on the weather.

> give case to malory
Protocol dictates that important assets should not be released until the
responsible agent confirms security precautions.

You could ask whether the safe house is ready or comment on the weather.

> a house
"The safe house, is it ready?" you inquire.

Malory scoffs: "Yes, there's a safe house. No, I won't reveal its location. This
is not a training exercise, why are you being so obtuse?"

Malory asks "What the devil are you up to? This whole exchange should be cut and
dried." He searches out your face in the gloom, looking for something.

You could comment on the weather.

> weather
"Damnable weather, isn't it?" You watch carefully to see how he reacts.
Sometimes a careful facade starts to crumble when faced with something
unexpected, such as a casual exchange in tense circumstances.

Are you satisfied? Hard to say in this business. People have secrets, wheels
within wheels. Still, it's time to hand him the package and hope for the best.

"It's London," he replies, "the weather is always damnable. I'm not here for
pleasantries." A solid response, but care must be taken.

You could ask what's wrong with pleasantries.

> drop case
The phrase "over my dead body" is usually an exaggeration. In this case, it is
chillingly accurate.

Begrudgingly, it's time to hand him the package and hope for the best.

"Perhaps field work has grown too taxing for you," he says, "we may need to have
a post-mortem back at the den." That sounds ominous.

> give it to him
You hand the briefcase to Malory and catch a look of something--greed? relief?
annoyance?--in his eyes. He disappears down the alley and is lost immediately in
the fog.

     *** The package has been delivered. Whether that's a triumph or a failure
will be known soon. ***

Would you like to:
     UNDO the last move,
     RESTORE a saved position,
     QUIT the program,
     or RESTART from the beginning?
> 
